client/src/components/layouts/global.css

```diff
@@ -12,83 +12,24 @@ body {
   background: var(--secondary-background);
 }
 
-/* This is to override Tailwind as it sets the font-weight to `bolder`.
-   TODO: Move this rule to @freecodecamp/ui. */
-b,
-strong {
-  font-weight: bold;
-}
-
-/* TODO: Move this rule to @freecodecamp/ui. */
-label {
-  display: inline-block;
-  max-width: 100%;
-  margin-bottom: 5px;
-  font-weight: bold;
-}
-
 /* TODO: Move this rule to @freecodecamp/ui. (Though some styles might need to be dropped). */
 legend {
-  display: block;
-  width: 100%;
-  padding: 0;
-  margin-bottom: 20px;
-  font-size: 21px;
-  line-height: inherit;
   color: #333;
-  border: 0;
```

Comment:
The base stylesheet doesn't have `border: 0` (I think it was missed), so let's keep it here to be safe.

I'll create an issue over UI library to add this.